eZ/Bundle/EzPublishRestBundle/Tests/Functional/ContentQueryTest.php

```diff
@@ -0,0 +1,380 @@
+<?php
+
+namespace eZ\Bundle\EzPublishRestBundle\Tests\Functional;
+
+use eZ\Bundle\EzPublishRestBundle\Tests\Functional\TestCase as RESTFunctionalTestCase;
+
+class ContentQueryTest extends RESTFunctionalTestCase
```

Comment:
1. Please add PhpDoc with brief description what this class tests.
2. I don't like naming, maybe let's call it `SearchViewTest`.